### ISSUE TITLE
Minify the bundled js

### ISSUE THREAD

**Issue by xuset** [xuset/planktos#1]
Integrate [uglify-js](https://www.npmjs.com/package/uglify-js) into the build process. If the inline source maps that browserify generates cannot be included in the minified js then it is probably best to have two build processes, one for production and the other for debug.

**Comment by xuset** [comment]
uglify-js always gives a parse error when it encounters an arrow function. From my brief look into uglify-js it seems it supports arrow functions so I'm not sure what's going on.

Stacktrace:
```Parse error at -:34,17
SyntaxError: Unexpected token: operator (>)
Error
    at new JS_Parse_Error (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:1547:18)
    at js_error (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:1555:11)
    at croak (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2094:9)
    at token_error (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2102:9)
    at unexpected (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2108:9)
    at expr_atom (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2635:9)
    at maybe_unary (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2797:19)
    at expr_ops (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2832:24)
    at maybe_conditional (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2837:20)
    at maybe_assign (eval at <anonymous> (/Users/xuset/code/planktos/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2861:20)
```

**Comment by xuset** [comment]
Closed in #16
